ddpui/management/commands/createorganduser.py

```
@@ -85,7 +92,14 @@ def handle(self, *args, **options):
             print(f"User {options['email']} already exists")
         user = User.objects.filter(email=options["email"]).first()
 
+        if user is None:
+            print(f"Error: Failed to create or fetch user '{options['email']}'")
+            sys.exit(1)
+
         if not OrgUser.objects.filter(org=org, user=user).exists():
+            if role is None:
+                print(f"Error: Role is None. This should not happen.")
```

Ruff (0.14.2)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)